stonesoup/types/tests/test_angle.py

```diff
@@ -23,10 +23,12 @@ def pytest_generate_tests(metafunc):
 
 scenario1 = ('bearing', {'Class': Bearing, 'func': mod_bearing})
 scenario2 = ('elevation', {'Class': Elevation, 'func': mod_elevation})
+scenario3 = ('longitude', {'Class': Longitude, 'func': mod_bearing})
```

Comment:
I thought you were planning to change these to `class_`. Is that still the plan?

Reply:
Correct, I just hadn't gotten around to doing it. Should be done now.